src/index.js

```diff
@@ -21,41 +54,11 @@ if (!window.cordova && window.location.protocol === 'file:') {
 
 // CSS.
 if (utils.device.isBrowserEnvironment) {
+  window.logs = debug;
```

Comment:
This used to be part of `debug.js`, but would be executed only when `isBrowserEnvironment` which is defined in `device.js`, which in turn depends on `debug.js`. This circular dependency caused issues.